stratum/hal/lib/common/common.proto

```diff
@@ -250,6 +252,59 @@ enum FecMode {
   FEC_MODE_AUTO = 3;
 }
 
+// The input optical power of this port in units of 0.01 decibel-milliwatt.
+// If the port is an aggregate of multiple physical channels, this attribute
+// is the total power or sum of all channels.
+//
+// The time-based values are the timestamps in nanoseconds relative to the
+// Unix Epoch (Jan 1, 1970 00:00:00 UTC)."
+message InputPower {
+  // The instantaneous value of the statistic.
+  ywrapper.Decimal64Value instant = 1;
```

Comment:
Is there a strong need to have this `Decimal64Value` type?

Current Stratum practice is to use a scalar type and have an implicit type/scale: https://github.com/stratum/stratum/blob/d026ea398ee8c5b6b5adfd9b2eea596446172115/stratum/hal/lib/phal/db.proto#L135

Without this all the adapter code could be dropped too.

Reply:
Unfortunately, gNMI doesn't support plain double: https://github.com/openconfig/gnmi/blob/master/proto/gnmi/gnmi.proto#L108.

It was agreed in Slack chat to introduce the new "Decimal64" type into the common.proto. Now, I've already pushed that for review.

Reply:
I just noticed that we convert `Decimal64` to `tai_float_t` which is a `float` type. 
https://github.com/Telecominfraproject/oopt-tai/blob/master/inc/taitypes.h#L54

I agreed to add `Decimal64` in Slack chat because I thought we need a high precision number in TAI. But looks like we only need `float` level precision. Do we have a strong need to have this type if the `float` type is enough?

Reply:
I was referring to the OpenConfig model, which considers the decimal type there.

Do we rely on the OpenConfig YANG model in this situation?
Or is it better to go with the "float" and cause the precision loss explicitly?

Reply:
Won't we lose precision anyway at some point? Yi suggest doing the conversion earlier (p4_service.cc probably) rather than later (TAIManager.cc) and save us the additional data type.

Reply:
Changed to
* `Decimal64` from the gNMI side
* and `float` from the Stratum side.

Please, review.